src/modules/Workspaces/WorkspacesLib/AppUtils.cpp

```diff
@@ -257,6 +259,18 @@ namespace Utils
 
                     if (appPathUpper.contains(installPathUpper))
                     {
+                        // check if the found app is the System Settings. If yes, update the install path to the exe path
+                        if (appPathUpper.ends_with(NonLocalizable::SystemSettingsPath))
+                        {
+                            return AppData{
+                                .name = NonLocalizable::SystemSettingsName,
+                                .installPath = appPath,
+                                .packageFullName = appData.packageFullName,
+                                .appUserModelId = appData.appUserModelId,
+                                .canLaunchElevated = appData.canLaunchElevated
+                            };
```

Comment:
```suggestion
                            auto settingsAppData = appData;
                            settingsAppData.installPath = appPath;
                            return settingsAppData;
```

Reply:
Let's only replace the path in case of future changes, such as a Settings app name change or adding more members in the AppData structure.

Reply:
Thanks, I agree.